CHANGELOG.md

```diff
@@ -2,6 +2,45 @@
 
 ## unreleased
 
+General:
+* Added more tests for the [crowdfunding example project](/examples/crowdfunding) using `@algo-builder/runtime`- Happy paths and Failing paths.
+* Added new command `algob test` (runs mocha in project root).
+* [breaking] Rename project `algorand-builder` to `algo-builder`.
+* Integrate user documentation with `jekyll`.
+* Moved OptIn methods from *DEPLOY* mode to *RUN* mode.
+* Added new algob commands:
+    * `algob unbox-template <name> <destination>` to quickly unbox a dapp template from `scale-it/algo-builder-templates`.
+    * `algob sign-multisig --account-name <acc> --file <input> --out <out-file>` to append user's signature to signed multisig file.
+* [breaking] Removed Algob prefix in deployer (eg. renamed `AlgobDeployer` to `Deployer`)
```

Comment:
let's move breaking changes to the section above, and let's remove "[breaking]" prefix.

Reply:
done